commands.js

```diff
@@ -1,73 +1,74 @@
-var exports = module.exports = {};
+const exports = module.exports = {};
 
-var emotes = {
-  igotit: 'I got it!',
-  taketheshot: 'Take the shot!',
-  defending: 'Defending...',
-  centering: 'Centering...',
-  omg: 'OMG!',
-  wow: 'Wow!',
-  closeone: 'Close one!',
-  no: 'Nooo!',
-  shi: '$#@!',
-  whoops: 'Whoops!',
-  sorry: 'Sorry!',
-  noproblem: 'No problem.',
-  thanks: 'Thanks!',
-  whatasave: 'What a save!',
-  greatpass: 'Great pass!',
-  niceshot: 'Nice shot!',
-  allyours: 'All yours.',
-  calculated: 'Calculated.',
-  goforit: 'Go for it!',
-  greatclear: 'Great Clear!',
-  holycow: 'Holy cow!',
-  inposition: 'In position.',
-  incoming: 'Incoming!',
-  mybad: 'My bad…',
-  myfault: 'My fault.',
-  needboost: 'Need Boost!',
-  niceblock: 'Nice Block!',
-  niceone: 'Nice one!',
-  noway: 'No Way!',
-  okay: 'Okay.',
-  oops: 'Oops!',
-  savage: 'Savage!',
-  sick: 'Siiiick!',
-  whataplay: 'What a play!',
-  whew: 'Whew.',
-  everybodydance: 'Everybody dance!',
-  gg: 'gg',
-  nicemoves: 'Nice moves.',
-  onemoregame: 'One. More. Game.',
-  rematch: 'Rematch!',
-  thatwasfun: 'That was fun!',
-  wellplayed: 'Well played.',
-  whatagame: 'What a Game!'
+const emotes = {
+    igotit: 'I got it!',
+    taketheshot: 'Take the shot!',
+    defending: 'Defending...',
+    centering: 'Centering...',
+    omg: 'OMG!',
+    wow: 'Wow!',
+    closeone: 'Close one!',
+    no: 'Nooo!',
+    shi: '$#@!',
+    whoops: 'Whoops!',
+    sorry: 'Sorry!',
+    noproblem: 'No problem.',
+    thanks: 'Thanks!',
+    whatasave: 'What a save!',
+    greatpass: 'Great pass!',
+    niceshot: 'Nice shot!',
+    allyours: 'All yours.',
+    calculated: 'Calculated.',
+    goforit: 'Go for it!',
+    greatclear: 'Great Clear!',
+    holycow: 'Holy cow!',
+    inposition: 'In position.',
+    incoming: 'Incoming!',
+    mybad: 'My bad…',
+    myfault: 'My fault.',
+    needboost: 'Need Boost!',
+    niceblock: 'Nice Block!',
+    niceone: 'Nice one!',
+    noway: 'No Way!',
+    okay: 'Okay.',
+    oops: 'Oops!',
+    savage: 'Savage!',
+    sick: 'Siiiick!',
+    whataplay: 'What a play!',
+    whew: 'Whew.',
+    everybodydance: 'Everybody dance!',
+    gg: 'gg',
+    nicemoves: 'Nice moves.',
+    onemoregame: 'One. More. Game.',
+    rematch: 'Rematch!',
+    thatwasfun: 'That was fun!',
+    wellplayed: 'Well played.',
+    whatagame: 'What a Game!'
 };
 
 exports.getCommand = function (command) {
   // Break out the command so we can parse it, and initialize the finalCommand to invalid.
-  var commands = command.split(" ");
-  var finalCommand = { type: "invalid", text: "", times: 0 };
+    const commands = command.split(' ');
+    const finalCommand = { type: 'invalid', text: '', times: 0 };
```

Comment:
The use of const here is valid, unless slack is running an older version of node that doesn't support const. You can set properties of a const, just not reassign the variable.